src/fetch.js

```diff
@@ -0,0 +1,17 @@
+let fetcher = {
```

Comment:
This looks good! Really convienent setup having the `fetch` stuff wrapped in an object.